operations/iuf/stages/management_nodes_rollout.md

```diff
@@ -218,3 +220,85 @@ or [Upgrade CSM and additional products with IUF](../workflows/upgrade_csm_and_a
             --initrd "s3://${S3_ARTIFACT_PATH}/initrd" \
             --params "${PARAMS}"
         ```
+
+## Upgrade NCN storage nodes into the customized image
+
+For the CSM upgrade from CSM 1.3 to CSM 1.4, the NCN storage node image does not change so there is no need to 'upgrade' to the customized storage image.
+It is recommended that NCN storage nodes are personalised by following the procedure in [management-nodes-rollout documentation](../workflows/management_rollout.md#3-execute-the-iuf-management-nodes-rollout-stage)
```

Comment:
We really want the customer to personalize the nodes, right?  Assuming so, I suggest you use stronger wording than "It is recommended...".

Reply:
Removed recommendation and told user to personalize the image